src/odb/src/db/dbGuide.cpp

```diff
@@ -110,6 +116,7 @@ dbIStream& operator>>(dbIStream& stream, _dbGuide& obj)
   stream >> obj.net_;
   stream >> obj.box_;
   stream >> obj.layer_;
+  stream >> obj.via_layer_;
```

Comment:
@maliberty Should I add the check for the schema version here? I believe the codeGenerator would remove manual changes here, right?

Reply:
I think you just need to add:
```
            "schema":"db_schema_db_guide_via_layer"
```
to the json file

Reply:
It worked, thanks very much!